interfaces/MIR100/tools/motion/assets/bundle.js

```diff
@@ -873,7 +873,7 @@ function clearPath() {
 document.body.appendChild(mainUI.domElement);
 
 /**
-* KineticARView contains all information, assets and logic for the threejs scene
+* ARScene contains all information, assets and logic for the threejs scene
 */
 var kineticARView = new _kineticARView.KineticARView();
```

Comment:
Pretty sure this should be ARScene based on the rename above

Reply:
yep will change. Thanks!